server/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19 AS build
+FROM golang:1.22 AS build
```

Comment:
If you want to upgrade go, regression testing is needed.

Reply:
@rot1024 it's related to `./../../../../go/pkg/mod/github.com/pelletier/go-toml/v2@v2.2.3/marshaler.go:11:2: package slices is not in GOROOT` 

https://github.com/pelletier/go-toml/issues/965

To whom should I discuss on the regression testing matter?

Reply:
@rot1024 reverted back to 1.19 for now